modules/manage/partials/remote-read-replicas.adoc

```diff
@@ -9,7 +9,7 @@ ifdef::env-kubernetes[]
 endif::[]
 ifndef::env-kubernetes[]
 :tiered-storage-link: manage:tiered-storage.adoc
-:data-archiving-link: manage:data-archiving.adoc
+:data-archiving-link: manage:tiered-storage.adoc#data-archiving
 endif::[]
 
 A Remote Read Replica topic is a read-only topic that mirrors a topic on a different cluster. Remote Read Replicas work with both xref:{tiered-storage-link}[Tiered Storage] and xref:{data-archiving-link}[archival storage].
```

Comment:
For Kubernetes, the second link goes to a section outside of Kubernetes. Is that correct?